Dockerfile

```diff
@@ -121,6 +124,7 @@ RUN sudo -H pip install --upgrade pip && \
     sudo -H pip install wheel tox && \
     sudo -H pip3 install --upgrade pip && \
     sudo -H pip3 install wheel tox && \
+    sudo -H pip3 install rpy2 && \
```

Comment:
We don't know to install `rpy2` here, should install by Airflow , not Airflow-CI

Reply:
I have deleted this line.